### DISCUSSION TITLE
File Uploads: Assign randomly generated file names but display original file name

### DISCUSSION THREAD

**Discussion by intrepidws** [filamentphp/filament#3996] [Help] [answered]
I'm using the basic `FileUpload` component, which assigns randomly generated file names. I'm also using the `storeFileNamesIn('attachment_file_names')` method to store the original file name. What I'd like to do now is have that original file name display in the image preview, rather than the randomly generated file name. Is that possible?

**Comment by danharrin** [comment, marked as answer]
Unfortunately I don't think there is a Filepond option to show a different file name than the one that the file is uploaded with...
